solarwinds_apm/apm_constants.py

```diff
@@ -17,6 +17,7 @@
 INTL_SWO_X_OPTIONS_KEY = "sw_xtraceoptions"
 INTL_SWO_X_OPTIONS_RESPONSE_KEY = "xtrace_options_response"
 INTL_SWO_DEFAULT_TRACES_EXPORTER = "solarwinds_exporter"
+INTL_SWO_DEFAULT_METRICS_EXPORTER = "otlp_proto_grpc"
```

Comment:
I'm wondering whose responsibility it should be to install `opentelemetry-exporter-otlp-proto-grpc`: APM Python or the user. If we do make `OTLPMetricExporter` a default that exports to SWO `otel.collector` by default, it should be APM Python. If not, the user perhaps?

Reply:
Oh hmm good question which I completely ignored with @raphael-theriault-swi's https://github.com/solarwindscloud/solarwinds-apm-js/pull/98. For non-lambda use case we're not likely to support exporting via OTLP right away, so in my mind bundling these exporters would be part of the custom lambda layer build and our regular distribution should not install them.

Reply:
Sounds good! Removed this constant and there won't be a default metrics exporter at least for now. `opentelemetry-exporter-otlp-proto-grpc` will be part of the Lambda layer, else users need to install it in their own environment.